src/components/Card.js

```diff
@@ -1,8 +1,22 @@
-import React from "react";
+import React, {useState} from "react";
+import DeleteSong from "./DeleteSong";
+import UpdateSong from "./UpdateSong";
 
 export const Card = ({ song }) => {
-    
+    const [isDeleteModalVisible, setDeleteModal] = useState(false);
+    const [isUpdateModalVisible, setUpdateModal] = useState(false);
+
+
+    const toggleDeleteModal = () => {
+      setDeleteModal(!isDeleteModalVisible);
```

Comment:
As a precaution in hooks, when the state update relies on the current state, it's best to use an updater function
```suggestion
      setDeleteModal(prevDeleteModalVisibility =>  !prevDeleteModalVisibility);
```